src/main/java/seedu/address/logic/Messages.java

```diff
@@ -35,21 +35,24 @@ public static String getErrorMessageForDuplicatePrefixes(Prefix... duplicatePref
      * Formats the {@code person} for display to the user.
      */
     public static String format(Person person) {
-        String telegramUsername = person.getTelegramUsername()
-                .toString() == null ? "" : person.getTelegramUsername().toString();
         final StringBuilder builder = new StringBuilder();
         builder.append(person.getName())
                 .append("; Phone: ")
                 .append(person.getPhone())
                 .append("; Email: ")
                 .append(person.getEmail())
                 .append("; Address: ")
-                .append(person.getAddress())
-                .append("; Telegram username: ")
-                .append(telegramUsername)
-                .append("; Tags: ");
-        person.getTags().forEach(builder::append);
+                .append(person.getAddress());
+
+        appendTelegramUsernameToMsg(person, builder);
+
         return builder.toString();
     }
 
+    private static void appendTelegramUsernameToMsg(Person person, StringBuilder builder) {
```

Comment:
Nice, so this handles the printing of the error message when the telegram username isn't there